lib/createsend/subscriber.rb

```diff
@@ -22,16 +22,18 @@ def self.get(auth, list_id, email_address, include_tracking_preference=false)
     end
 
     # Adds a subscriber to a subscriber list.
-    def self.add(auth, list_id, email_address, name, custom_fields, resubscribe, 
-      consent_to_track, restart_subscription_based_autoresponders=false)
+    def self.add(auth, list_id, email_address, name, mobile_number, custom_fields, resubscribe, 
```

Comment:
Should we put this in the last fields, and potentially with default values so that it won't break existing 'add' functionality?